sites/fast-component-explorer/app/utilities/configs/slider.ts

```diff
@@ -36,36 +37,36 @@ const sliderConfig: ComponentViewConfig<SliderProps> = {
                     {
                         id: sliderLabelSchema.id,
                         props: {
-                            valuePositionBinding: 0,
-                            label: "low",
+                            valuePositionBinding: SliderTrackItemAnchor.totalRangeMin,
```

Comment:
There is a large effort to update this entire app on a feature branch. If this is not urgent can we hold off?

Reply:
which pr / issue / branch is this pending on?

Reply:
Jane said `feature/update-explorer-site-to-tooling-v2-and-web-components`. I haven't looked into making the change there as well yet.

Reply:
I forgot to post that here, but a new wrinkle was introduced just now where we're discussing placing only web components in the component explorer. Let's take this discussion to discord and then we can update this PR with the resolution.